src/openrct2-ui/ride/Construction.cpp

```diff
@@ -24,6 +24,169 @@ using namespace OpenRCT2::TrackMetaData;
+        TrackElemType::LeftTwistUpToDown,
+        TrackElemType::RightTwistDownToUp,
+        TrackElemType::RightTwistUpToDown,
+        TrackElemType::LeftFlyerTwistUp,
+        TrackElemType::LeftFlyerTwistDown,
+        TrackElemType::RightFlyerTwistUp,
+        TrackElemType::RightFlyerTwistDown,
+
+        // Corkscrews
+        TrackElemType::LeftCorkscrewUp,
+        TrackElemType::LeftCorkscrewDown,
+        TrackElemType::RightCorkscrewUp,
+        TrackElemType::RightCorkscrewDown,
+        TrackElemType::LeftFlyerCorkscrewUp,
+        TrackElemType::LeftFlyerCorkscrewDown,
+        TrackElemType::RightFlyerCorkscrewUp,
+        TrackElemType::RightFlyerCorkscrewDown,
+        TrackElemType::LeftLargeCorkscrewUp,
+        TrackElemType::LeftLargeCorkscrewDown,
+        TrackElemType::RightLargeCorkscrewUp,
+        TrackElemType::RightLargeCorkscrewDown,
+
+        // Helixes
+        TrackElemType::LeftHalfBankedHelixUpSmall,
+        TrackElemType::RightHalfBankedHelixUpSmall,
+        TrackElemType::LeftHalfBankedHelixDownSmall,
+        TrackElemType::RightHalfBankedHelixDownSmall,
+        TrackElemType::LeftHalfBankedHelixUpLarge,
+        TrackElemType::RightHalfBankedHelixUpLarge,
+        TrackElemType::LeftHalfBankedHelixDownLarge,
+        TrackElemType::RightHalfBankedHelixDownLarge,
+        TrackElemType::LeftQuarterBankedHelixLargeUp,
+        TrackElemType::RightQuarterBankedHelixLargeUp,
+        TrackElemType::LeftQuarterBankedHelixLargeDown,
+        TrackElemType::RightQuarterBankedHelixLargeDown,
+        TrackElemType::LeftQuarterHelixLargeUp,
+        TrackElemType::RightQuarterHelixLargeUp,
+        TrackElemType::LeftQuarterHelixLargeDown,
+        TrackElemType::RightQuarterHelixLargeDown,
+
+        // Zero-G Rolls
+        TrackElemType::LeftZeroGRollUp,
+        TrackElemType::LeftZeroGRollDown,
+        TrackElemType::RightZeroGRollUp,
+        TrackElemType::RightZeroGRollDown,
+        TrackElemType::LeftLargeZeroGRollUp,
+        TrackElemType::LeftLargeZeroGRollDown,
+        TrackElemType::RightLargeZeroGRollUp,
+        TrackElemType::RightLargeZeroGRollDown,
+
+        // Barrel Rolls
+        TrackElemType::LeftBarrelRollUpToDown,
+        TrackElemType::LeftBarrelRollDownToUp,
+        TrackElemType::RightBarrelRollUpToDown,
+        TrackElemType::RightBarrelRollDownToUp,
+
+        // Tower
+        TrackElemType::TowerBase,
+        TrackElemType::TowerSection,
+
+        // Boosters
+        TrackElemType::Booster,
+        TrackElemType::DiagBooster,
+
+        // Brakes
+        TrackElemType::Brakes,
+        TrackElemType::DiagBrakes,
+        TrackElemType::Down25Brakes,
+        TrackElemType::BlockBrakes,
+        TrackElemType::DiagBlockBrakes,
+
+        // Photo sections
+        TrackElemType::OnRidePhoto,
+
+        // River Rapids
+        TrackElemType::Waterfall,
+        TrackElemType::Rapids,
+        TrackElemType::Watersplash,
+        TrackElemType::Whirlpool,
+
```

Comment:
`// Miscellaneous` perhaps?

Reply:
We can adjust the comments once the list is more-or-less complete, heh.

Reply:
That's fair! Didn't realize you were still working on it haha